storageFormats/eventMetadata.h

```diff
@@ -112,6 +115,31 @@ namespace rpwa {
 		return metadata.print(out);
 	}
 
+
+	/// \brief Handles additional saved variables of event data
+	class additionalVariables {
+	public:
+		additionalVariables() {}
+		additionalVariables(const additionalVariables&) = delete;
+		additionalVariables& operator = (const additionalVariables&) = delete;
+		double operator [](const std::string& label) const {return _additionalSavedVariables.at(label);}
+
+		bool setBranchAddresses(const eventMetadata& metaData);
+
+		const std::vector<std::string>& labels() const {return _labels;}
+
+		/***
+		 * \brief checks wether the current additional variables are within the boundaries
+		 *
+		 * x_i in [lower_i, upper_i) for all variables i in the given boundaries
+		 */
+		bool inBoundaries(const rpwa::multibinBoundariesType& multibinBoundaries)const;
+
+	private:
+		std::map<std::string, double > _additionalSavedVariables;
+		std::vector<std::string> _labels;
```

Comment:
Is it necessary to duplicate the information already stored in `_additionalSavedVariables`?